packages/services/src/epcisEvents/types.ts

```diff
@@ -59,30 +59,28 @@ export interface IInputItems {
   productClass: string;
 }
 
-export interface ITransformationEvent extends IContext {
+export interface ITraceabilityEvent {
```

Comment:
I saw this interface used for almost all events, should we refactor processTransformationEvent to use this? 
<img width="999" alt="image" src="https://github.com/user-attachments/assets/af2801e0-5ff5-41fc-9947-68355c98176c">

Reply:
No, because the transformation event extends from the traceability event and has some additional data